src/js/index.html

```diff
@@ -10,11 +10,34 @@ <h1>Hello World</h1>
     <div id="TL"/>
     <script>
       function getTL(){
+        document.getElementById("TL").innerHTML = "";
         const xmlHttp = new XMLHttpRequest();
         xmlHttp.onreadystatechange = function() {
           if(xmlHttp.readyState === XMLHttpRequest.DONE){
             if(xmlHttp.status === 200){
-              document.getElementById("TL").innerHTML = xmlHttp.responseText;
+              const timelineList = JSON.parse(xmlHttp.responseText);
+              var i;
```

Comment:
this var i should not be needed. Fix it so the loop works without this external declaration